index.html

```diff
@@ -579,19 +580,15 @@ <h1>Trust Model</h1>
 risk assessment of the verifier.
         </li>
         <li>
-All entities trust the <a>identifier registry</a> to be incorruptible and
-to be a correct record of which identifiers belong to which <a>entities</a>.
+All entities trust the <a>Verifiable data repository</a> to be un-corruptible and
+to be a correct record of which data is owned (and can be updated) by which
+<a>entities</a>.
         </li>
         <li>
 The <a>subject</a> trusts the <a>issuer</a> to issue true (i.e., not false)
 <a>credentials</a> about the subject, and to revoke them quickly when
 appropriate.
         </li>
-        <li>
-The <a>holder</a> trusts the <a>repository</a> to store the
```

Comment:
This shouldn't be removed. This is talking about the holder's 'credential repository'... not the cryptographic material registry.

Reply:
Thanks, added back